### ISSUE TITLE
Has this been abandoned?

### ISSUE THREAD

**Issue by clehene** [google/startup-os#581]
Please update status, and let anyone bumping into this know whether it's in maintenance mode.

**Comment by oferb** [comment]
Hey @clehene!
I was the project lead for this effort, and have since left Google.
I don't know what the current status of the project it, but, my guess is that it's not worked on.

**Comment by clehene** [comment]
@oferb thank you! 
That's unfortunate. Good luck on your new projects!

**Comment by oferb** [comment]
Thanks!!

**Comment by zoidyzoidzoid** [comment]
It would be awesome if we could maybe have a fork of this that we updated, since I think it's an awesome example.

**Comment by oferb** [comment]
Go for it!

**Comment by zoidyzoidzoid** [comment]
Forked to https://github.com/startup-os/startup-os

**Comment by clehene** [comment]
> Forked to https://github.com/startup-os/startup-os

What’s the plan, @zoidbergwill? Will you start maintaining it?

**Comment by zoidyzoidzoid** [comment]
> > Forked to https://github.com/startup-os/startup-os
> 
> What’s the plan, @zoidbergwill? Will you start maintaining it?

Yeah, hopefully add a few other people keen to contribute and then we maintain it?

**Comment by oferb** [comment]
May the Fork be with you!

![yoda](https://user-images.githubusercontent.com/5137671/86038640-e4aad000-ba49-11ea-9b91-28b6afe277eb.jpeg)

**Comment by zoidyzoidzoid** [comment]
Added you as a contributor, @oferb.
